niceman/tests/utils.py

```diff
@@ -403,6 +403,32 @@ def newfunc(*args, **kwargs):
         check_and_raise()
 
 
+def skip_if_no_docker_container(container_name='testing-container'):
+    """Test decorator that will skip a test if the Docker container the test is
+    going to connect to is not running in the Docker engine.
+    
+    Parameters
+    ----------
+    container_name : str
+        Name of the container that needs to be running for the test to work.
+    
+    Returns
+    -------
+    func
+        Decorator function
+    
+    Raises
+    ------
+    SkipTest
+    """
+    def decorator(func):
+        stdout, _ = Runner().run(['docker', 'ps'])
+        if container_name not in stdout:
```

Comment:
very weak check since could be a partial match. Please TODO @mjtravers  - make matching by the value of the second column